x-pack/plugins/security_solution/public/resolver/test_utilities/simulator/index.tsx

```diff
@@ -265,6 +282,13 @@ export class Simulator {
     return this.resolveWrapper(() => this.domNodes(`[data-test-subj="${selector}"]`));
   }
 
+  /**
+   * Given a `role`, return DOM nodes that have it. Use this to assert that ARIA roles are present as expected.
+   */
+  public domNodesWithRole(role: string): ReactWrapper {
```

Comment:
In order to test our ARIA compliance we need to select nodes by their `role` attribute.

Reply:
this will be useful - thanks.